docs/plugins/analysis-kuromoji.asciidoc

```diff
@@ -12,10 +12,10 @@ include::install_remove.asciidoc[]
 
 The `kuromoji` analyzer consists of the following tokenizer and token filters:
 
+* `cjk_width` char filter
```

Comment:
I don't think Elasticsearch has a character filter named `cjk_width`, only a token filter. Maybe we could refer to this as "`CJKWidthCharFilter` from Lucene" ?

Also we could change the sentence above to say "The `kuromoji` analyzer uses the following analysis chain:" Currently it only mentions tokenizers and token filters.

Reply:
Good catch!
I forgot about the sentence above.